src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -1094,6 +1098,79 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
             return successful && File.Exists(fullDestPath);
         }
 
+        /// <summary>
+        /// Checks if the file at the given path is an MSFZ file by checking for the "Microsoft MSFZ Container" header
+        /// </summary>
+        private bool IsMsfzFile(string filePath)
+        {
+            try
+            {
+                if (!File.Exists(filePath))
+                    return false;
+
+                const string msfzHeader = "Microsoft MSFZ Container";
+                var headerBytes = Encoding.UTF8.GetBytes(msfzHeader);
+
+                using (var stream = File.OpenRead(filePath))
+                {
+                    if (stream.Length < headerBytes.Length)
```

Comment:
From a stylistic perspective, we use `{}` around `if` blocks.  Please update the style throughout the PR.